### DISCUSSION TITLE
`FlxCamera.defaultCameras` is deprecated, use `FlxG.cameras.setDefaultDrawTarget` instead

### DISCUSSION THREAD

**Discussion by khiodev** [HaxeFlixel/flixel#2484] [Q&A]
I am trying to compile my mod to html5 but it keeps giving me the error: 
`source/DiffOverview.hx:66: characters 3-39 : Warning : "FlxCamera.defaultCameras" is deprecated, use "FlxG.cameras.setDefaultDrawTarget" instead
source/GameplayCustomizeState.hx:81: characters 3-39 : Warning : "FlxCamera.defaultCameras" is deprecated, use "FlxG.cameras.setDefaultDrawTarget" instead
source/PlayState.hx:437: characters 3-39 : Warning : "FlxCamera.defaultCameras" is deprecated, use "FlxG.cameras.setDefaultDrawTarget" instead
source/Replay.hx:103: characters 16-35 : Warning : Usage of this field is deprecated
source/StageDebugState.hx:104: characters 3-39 : Warning : "FlxCamera.defaultCameras" is deprecated, use "FlxG.cameras.setDefaultDrawTarget" instead`

**Comment by Geokureli** [comment]
Those are warnings, not errors they won't prevent your game from building successfully. It's suggesting a new way to do what you want, because future updates may remove the old way. Show us what you're trying to do with cameras and we can help you
